cfbs/git.py

```diff
@@ -189,7 +197,7 @@ def git_check_tracked_changes(scope=["all"]):
         else:
             lines = result.stdout.decode("utf-8").split("\n")
             changes = [line.strip().split(" ")[1] for line in lines if line]
-            should_commit = any(i in changes for i in scope)
+            should_commit = _get_should_commit(changes, scope)
```

Comment:
You can use `itertools.product()` it test all the files against each other
```python
>>> from itertools import product
>>> list(product(["1", "2", "3"], ["A", "B", "C"]))
[('1', 'A'), ('1', 'B'), ('1', 'C'), ('2', 'A'), ('2', 'B'), ('2', 'C'), ('3', 'A'), ('3', 'B'), ('3', 'C')]
```

E.g.,
```python
should_commit = any(os.path.samefile(a, b) in itertools.product(changes, scope))
```

But what you did also OK ;)